src/workspaces/common/state/useWorkspaces.ts

```diff
@@ -25,7 +24,7 @@ const defaultFieldsArgs: FieldsArg = [
  */
 export const useWorkspaces = (fieldsArg?: FieldsArg, stringAttributeMaxLength?: number): UseWorkspacesResult => {
   const signal = useCancellation();
-  const { reportError } = withErrorReporter(useNotificationsFromContext());
+  const { reportError } = useNotificationsFromContext();
```

Comment:
Why the removal of `withErrorReporter` here?

Reply:
the core change that enabled full decoupling of errors.ts and notifications was to have the NotificationsProvider returned by useNotificationsFromContext to now include the reportError, withErrorReporting methods in its interface that previously had come through withErrorReporter helper.

the mechanics of what used to be withErrorReporter helper are now pre-backed into the constructed NotificationsProvider injected into react context at the root of app visual tree in Main.ts  There we use terraUINotifications export from what remains of src/libs/error.ts where the construction is done.

Reply:
Ah, I see. I looked through more of the code and understand now.